### ISSUE TITLE
Internationalization (i18n) for ant-icons / aria-label

### ISSUE THREAD

**Issue by BenICE** [ant-design/ant-design-icons#478]
###  What problem does this feature solve?

ant-icons are often used in the ant components -> e.g. the Drawer is using the ant icon for close. This is fine and works well, but we have no possibility to easily overwrite the aria-label of the icon.

The problem arises if your website language is not English, but another. The aria-label on the icon is English so the screenreader will read out "Close".

First it is the wrong language and you cannot even understand it properly as the screenreader will try to pronounce it in the website language.

### What does the proposed idea look like?

I guess the contextProvider need to be used to check the current language and then use it to translate the aria-label.

(I could add the German translations, if you wish to go ahead with this.)
